repos/system_upgrade/el7toel8/models/grubenv.py

```diff
@@ -0,0 +1,13 @@
+from leapp.models import Model
+from leapp.topics import SystemFactsTopic
+
+
+class HybridImageDetected(Model):
+    """
+    Model used for instructing Leapp to convert "grubenv" symlink
+    into a regular file in case of hybrid (BIOS/EFI) images using BIOS
+    on Azure.
+
+    This model will likely be removed in the future.
+    """
```

Comment:
Would't make sense to mark the mode to be deprecated in case it's expected it will be removed in future (regarding the docstring).

Reply:
Realized we will have to keep it anyway...